Gemfile

```diff
@@ -74,12 +74,11 @@ gem 'newrelic_rpm'
 gem 'twitter-typeahead-rails'
 gem 'blacklight_range_limit', '~> 7.0'
 gem 'redis', '~> 5.0'
-gem 'rsolr'
```

Comment:
I don't understand how this change supports deploying. Can you split this out to a separate PR?

Reply:
Ah, I see it's just deduplicating.